src/main/resources/db/changelog/db.changelog-master.xml

```diff
@@ -8,4 +8,5 @@
     <include file="db/changelog/db.changelog-RDM-2761-UserProfileAuditTable.xml"/>
     <include file="db/changelog/db.changelog-RDM-3853.xml"/>
     <include file="db/changelog/db.changelog-RDM-5499.xml"/>
+    <include file="db/changelog/db.changelog-RDM-9052.xml"/>
```

Comment:
We might need to remove this RDM-9052 change? @henrykcgi

Reply:
No Not really.. it is the fix and required.